### DISCUSSION TITLE
weibull and pn['pore.seed']

### DISCUSSION THREAD

**Discussion by Devin-one** [PMEAL/OpenPNM#2619] [Q&A]
Hello Professor, I am very interested in the Weibull distribution formula in your doctoral thesis. I wonder if `X `in the formula is the `pn['pore.seed']`?
![图片](https://user-images.githubusercontent.com/93040662/200220880-0b3faa4d-0c96-45fb-8bdc-7c9025dbc07f.png)

**Comment by jgostick** [comment]
I need to apologize for the late response here. To answer your question, yes, x is pore seed, BUT that formula was from a long time ago, so it may not be compatible with the OpenPNM way.
